tests/config/config.travis-ci.php

```diff
@@ -2,7 +2,7 @@
 
 $LRSs = [
     [
-        'endpoint' => 'https://cloud.scorm.com/tc/0CKX3A0SF2/sandbox/',
+        'endpoint' => 'https://cloud.scorm.com:443/tc/0CKX3A0SF2/sandbox/',
```

Comment:
I'm okay with that, but does that leave us open to failures when there *isn't* a port?